.github/workflows/tests.yml

```diff
@@ -18,6 +18,9 @@ jobs:
       matrix:
         python-version: ["3.7", "3.8", "3.9", "3.10", "3.11", "3.12"]
         os: ["macos-latest", "windows-latest", "ubuntu-latest"]
+        exclude:
+          - python-version: "3.7"
+            os: "macos-latest"
```

Comment:
Let's just leave this out for now -- I'd rather merge this with a broken CI (or wait a little longer for GHA to fix it) than have a matrix exclude rule that I'll likely forget about :slightly_smiling_face: